partners/scientific-communities.md

```diff
@@ -6,29 +6,34 @@
 
 About Our Partnerships <self>
 JOSS <joss>
+Astropy <astropy>
 Pangeo <pangeo>
```

Comment:
Great to see this list growing!

Reply:
@namurphy i'm excited about it too!! we also just spoke with scverse!!